mantle/cmd/ore/aws/upload.go

```diff
@@ -91,6 +94,9 @@ func init() {
 	cmdUpload.Flags().BoolVar(&uploadIMDSv2Only, "imdsv2-only", false, "enable IMDSv2-only support")
 	cmdUpload.Flags().StringVar(&uploadVolumeType, "volume-type", "gp3", "EBS volume type (gp3, gp2, io1, st1, sc1, standard, etc.)")
 	cmdUpload.Flags().StringVar(&uploadX86BootMode, "x86-boot-mode", "uefi-preferred", "Set boot mode (uefi-preferred, uefi)")
+	cmdUpload.Flags().BoolVar(&uploadCreateWinLIAMI, "winli", false, "Create a Windows LI AMI")
+	cmdUpload.Flags().StringVar(&uploadWinLIwindowsServerAMI, "windows-ami", "", "Windows Server AMI used to create a Windows LI AMI")
+	cmdUpload.Flags().StringVar(&uploadWinLIInstanceType, "winli-instance-type", "", "ec2 instance type used to create a Windows LI AMI (default: t2.large)")
```

Comment:
```suggestion
	cmdUpload.Flags().StringVar(&uploadWinLIInstanceType, "winli-instance-type", "t2.large", "ec2 instance type used to create a Windows LI AMI")
```

I think you can just set a default this way and you won't have to do the conditional logic below.